src/System.Runtime.Extensions/tests/CustomAttributesTestApp/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Reflection;
+using System.Threading;
+
+namespace CustomAttributesTestApp
```

Comment:
Nit: A better name for this may be `TargetFrameworkNameTestApp`

Reply:
I used that name as in future we might want to test other attributes here as well.

Reply:
FWIW, I prefer @jkotas' naming suggestion.  If in the future we want to expand it to encompass other things, it can be renamed then as desired.